frontend/src/styles/globals.css

```diff
@@ -1,3 +1,13 @@
+:root{
+  --black: #000;
+  --white: #fff;
+  --main-primary: #66d9e8;
+
+  --main-secondary: #0b7285;
+  --main-light-secondary: #0c8599;
+  --main-light-secondary-2: #1098ad;
+}
+
```

Comment:
- Remove this color variable declarations from here and add to `.js` file.
- Reference from branch `feature/left-drawer`